README.md

```diff
@@ -303,5 +322,5 @@ from the root directory of the repository.
 Furthermore, all tests in the ```tests``` directory will be run upon pushing changes by a github action. Failure in any of the tests will also reject the push/PR.
 
 # Contact
-If you are interested in machine learning models for LAM, have questions about our implementation or ideas for extending it, feel free to get in touch.
-You can open a github issue on this page, or (if more suitable) send an email to [joel.oskarsson@liu.se](mailto:joel.oskarsson@liu.se).
+If you are interested in machine learning models for LAM, have questions about the implementation or ideas for extending it, feel free to get in touch.
+You can join the [slack channel](https://join.slack.com/t/ml-lam/shared_invite/zt-2t112zvm8-Vt6aBvhX7nYa6Kbj_LkCBQ), open a github issue on this page, or (if more suitable) send an email to [joel.oskarsson@liu.se](mailto:joel.oskarsson@liu.se).
```

Comment:
Be careful when publishing this kind of information. I was involved in another project where we couldn't add any new member to the slack channel because bots had spammed the slack join link.

Reply:
That's a good point :scream: Currently, members from a curated list of domains (unis, met services mostly) can join without invite. The link you are about to post can be used 400-times. If abused, the link can be deactivated and a new one generated. All bots, must be removed manually by the workspace owner. That sounds annoying...

Alternatively, we can set up admin admission via notification in one of our slack channels (https://slack.com/help/articles/115004854783-Require-admin-approval-for-workspace-invitations). Since, we are all admins this could be a good solution. What do you think?

Reply:
Yes thanks for pointing this out :open_hands: I did not know if bots would be an issue with slack, but clearly it can be then. 

I don't really get what will happen if you set up this admin approval. I guess you will not immediately join through the link then, but send a request to join? And then any admin can accept it. In that case we would definitely not want the option where requests are sent to all admins via Slackbot, as then everyone will be spammed by every request. But one can set it to post the requests to a specific channel, which could work fine?

Reply:
@sadamov, Yes I think that might mitigate the problem. If I remember well, the issue was that the bots activate the join link, but never actually join the slack channel. Resulting in 400 _people_ in limbo for activation. Which blocks any new user from joining. 
Using admin rights to actively reject them might solve the issue. It will also give you an idea if the link is getting abused, so that you can remove it if it becomes problematic.

Reply:
Maybe a good idea to make a specific channel for it? Or otherwise make a post to clarify that invite requests will show up in random, so people are not confused about it (e.g. if someones colleague joins it might not be obvious where to find the message to accept the request).

Reply:
I'm thinking that I'll also clarify in the readme the procedure for joining slack, so people aren't deterred when you have to request to join